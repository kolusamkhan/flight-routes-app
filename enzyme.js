const Adapter = require('enzyme-adapter-react-17-updated')

require('enzyme').configure({adapter: new Adapter()})