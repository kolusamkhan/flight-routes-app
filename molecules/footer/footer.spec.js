import 'jsdom-global/register';
import * as React from 'react';
import { act } from 'react-dom/test-utils';
import expect from 'expect';
import { shallow, mount, render } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';
import Footer from './index';

describe('Footer', () => {
  describe('Footer component', () => {
    it('should render without throwing an error', function () {
      const wrapper = shallow(<Footer />);
      expect(shallowToJson(wrapper)).toMatchSnapshot();
    })
    
    it('should render Footer content', function () {
        const wrapper =  mount(<div><Footer /></div>);
        expect(wrapper.text()  === "Powered by ").toBeTruthy()
      })
  })  
})