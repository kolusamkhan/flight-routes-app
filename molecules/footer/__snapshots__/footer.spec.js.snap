// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer Footer component should render without throwing an error 1`] = `
<footer
  className="footer"
>
  <a
    href="https://vercel.com?utm_source=create-next-app&utm_medium=default-template&utm_campaign=create-next-app"
    rel="noopener noreferrer"
    target="_blank"
  >
    Powered by
     
    <span
      className="logo"
    >
      <Image1
        alt="Vercel Logo"
        height={16}
        src="/vercel.svg"
        width={72}
      />
    </span>
  </a>
</footer>
`;
